src/pages/NewUser/NewUser.jsx

```diff
@@ -0,0 +1,61 @@
+import Layout from "../../components/Layout/Layout";
+import Button from "../../components/Button/Button";
+import "./NewUser.scss";
+import logo from "../../assets/images/AppLogo.svg";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+const NewUser = () => {
+  const [buttonText, setButtonText] = useState("Get Started");
+  const [content, setContent] = useState(
+    " Bringing local business to local people"
+  );
+
+  const buttonClicked = () => {
+    if (buttonText == "Get Started") {
+      setButtonText("Continue");
+      setContent(
+        "No matter where you are in the world Local shows you the best local bars, cafes, restaurants and shops"
+      );
+    } else if (buttonText == "Continue") {
+      setButtonText("Sign Up");
+      setContent(
+        "Discover new places and curate your own personal list. Be a local anywhere"
+      );
+    }
+  };
+
+  const buttonJSX = (
+    <div onClick={buttonClicked}>
```

Comment:
you have a button here, can you add this onclick to that button and remove the div?

Reply:
done